locales/en/smart-events.json

```diff
@@ -40,7 +41,8 @@
       "ready": "Ready",
       "creating": "Creating",
       "deleting": "Deleting",
-      "failed": "Failed"
+      "failed": "Failed",
+      "preparing": "Preparing"
```

Comment:
Why did you add the `preparing` translation here? :thinking: 
I couldn't find any usages.

Reply:
Probably, at some point, I was introducing it for the filter. BTW, removing it!